google/resumable_media/_download.py

```diff
@@ -466,3 +472,28 @@ def get_range_info(response, get_headers, callback=_helpers.do_nothing):
         int(match.group(u'end_byte')),
         int(match.group(u'total_bytes'))
     )
+
+
+def check_for_zero_content_range(response, get_status_code, get_headers):
+    """ Validate if response status code is 416 and content range is zero.
+
+    This is the speacial case for handling zero bytes files.
```

Comment:
Fix the grammar and typo:
`This is special case for handling zero bytes files.`

Reply:
My bad, will do it.

Reply:
Done.